walkthroughs/week-5-tdd/project/src/main/java/com/google/sps/FindMeetingQuery.java

```diff
@@ -15,9 +15,77 @@
 package com.google.sps;
 
 import java.util.Collection;
+import java.util.Collections;
+import java.util.Set;
+import java.util.List;
+import java.util.ArrayList;
+import com.google.sps.TimeRange;
+import java.util.Comparator;
+import com.google.sps.Event;
 
 public final class FindMeetingQuery {
```

Comment:
At Google we expect all classes to have a javadoc style comment which describes what that purpose of the class is. It's a good habit you should start building, and I think we will try to enforce for the final project.